src/code/components/chromosome.js

```diff
@@ -83,10 +77,11 @@ const ChromosomeView = ({chromosome, ChromosomeImageClass=ChromosomeImageView, u
     <div className="geniblocks chromosome-container" onClick={ handleSelect } >
       <div className={ containerClass }>
         <div className="chromosome-allele-container" id={chromId} style={displayStyle}>
-          <ChromosomeImageClass isSelectedEmpty={isSelectedEmpty} small={small} empty={empty} bold={selected} chromosomeDescriptor={getChromosomeDescriptor(chromosome)}/>
-          { allelesContainer }
+          <ChromosomeImageClass small={small} empty={empty} bold={selected} chromosomeDescriptor={chromosomeDescriptor}/>
+        </div>
+        <div className="labels">
```

Comment:
I'm always leaving the labels div in here, although for empty chromosomes the div will be empty. This helps with formatting, since now chromosomes have the same length even as they gain or lose labels